package.json

```diff
@@ -33,6 +33,7 @@
   },
   "scripts": {
     "start": "PORT=3001 react-scripts start",
+    "start:local": "env-cmd -f .env.local react-scripts start",
```

Comment:
There is no .env.local

Reply:
ah de .gitignore negeerde die file name. pas het aan